### PR TITLE
refactor(components): Adjust banner alignment

### PR DESCRIPTION
**Pull request by rodrigoeidelvein**
## Changes

- Changed banner alignment

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/components/src/Banner/Banner.css [hunk 1]
```diff
@@ -10,7 +10,7 @@
   padding: var(--flash--inset);
   color: var(--color-black);
   background-color: var(--color-grey--lightest);
-  align-items: flex-start;
+  align-items: center;
```
Comment (chris-at-jobber) [inline packages/components/src/Banner/Banner.css:13]: @rodrigoeidelvein this breaks the alignment from what we expect / what was previously there

You can see this pushes the dismiss button to the center of the banner:
image

It should remain aligned to the top:
image

**Review by chris-at-jobber — CHANGES_REQUESTED** [review]
Needs to solve the problem without breaking existing reflow behaviour

**Comment by chris-at-jobber** [comment]
Wide looks good ✅ 
image

Narrow looks good ✅ 
image
 
There's this intermediate break where I think the styling for a "narrow" banner is added, but the content is not long enough to wrap:
image

I wouldn't call this a blocker since we already see this behaviour (see example), but wanted to flag it.
image

**Review by chris-at-jobber — APPROVED** [review]
The changes look good to me. We'll definitely want to test this one "in the wild" with more complex usages, but this seems like a reasonable way to encapsulate the alignment into separate containers. It looks like you may have inadvertently also resolved an issue where custom child content could break!

### Before
Bullet list reflows on same line as paragraph, not good:
image

### After
Bullet list is beneath paragraph, as expected!
image

**Comment by rodrigoeidelvein** [comment]
@chris-at-jobber that was all calculated 🤣

**Pull request merged**